lib/containers/k8s.pm

```diff
@@ -72,6 +72,11 @@ sub install_k3s {
     script_retry("kubectl get serviceaccount default -o name", delay => 60, retry => 3);
     # Await k3s to be ready and the api is accessible
     script_retry("kubectl get pods --all-namespaces", delay => 60, retry => 3);
+    assert_script_run("systemctl is-active k3s");
+    assert_script_run("kubectl api-resources");
```

Comment:
If this is for debugging purposes, I would consider putting it into a `record_info`, similar to lines 80 and 81.

Reply:
Done